src/ECS/utilities/Zipper/GenericZipper.hpp

```diff
@@ -8,9 +8,41 @@
 #ifndef GENERIC_ZIPPER_HPP
     #define GENERIC_ZIPPER_HPP
 
-    #include <algorithm>
     #include <cstddef>
-    #include <vector>
+    #include <tuple>
+    #include <utility>
+
+template<typename T>
```

Comment:
Could you add documentation to the following functions ? So that we know what they do ?

Reply:
Sure, I intended to do it but later forgot about it